bin/yaml/libraries.yaml

```diff
@@ -1025,7 +1025,7 @@ libraries:
         targets:
         - trunk
         type: github
-      beman_iterator:
+      beman_iterator_interface:
```

Comment:
@junlarsen , quick question (I'm also from Beman org) - the library name is actually `beman.iterator_interface`.


<img width="1229" alt="image" src="https://github.com/user-attachments/assets/815aa9a6-fcfc-476a-9723-e329646a2e49">

Can we change here such we can see `beman.iterator_interface` in UI?
<img width="1229" alt="image" src="https://github.com/user-attachments/assets/35e0f2d4-9f27-47b6-b76f-3e4595752fb9">

Reply:
Those are the `.name` entries here https://github.com/compiler-explorer/compiler-explorer/blob/main/etc/config/c%2B%2B.amazon.properties#L3867 - you can freely change them

Reply:
Cool. So it's handled from the other repo. Cool!

@partouf , can we merge this PR?